projects/storefrontlib/src/cms-components/user/register/register.component.ts

```diff
@@ -45,14 +47,25 @@ export class RegisterComponent implements OnInit, OnDestroy {
     { validator: this.matchPassword }
   );
 
+  /**
+   * @deprecated since 1.1.0
+   *
+   * TODO(issue:4237) Register flow
+   */
   constructor(
-    private auth: AuthService,
-    private authRedirectService: AuthRedirectService,
-    private userService: UserService,
-    private globalMessageService: GlobalMessageService,
-    private fb: FormBuilder
+    protected userService: UserService,
```

Comment:
Provide 1.0 compatible constructor. Currently changing order of services is a breaking change.

Reply:
I had some problems with constructor overload and adding AND removing deps. We have to test this one out